test/types/range/open/openRanges-both.good

```diff
@@ -0,0 +1,3 @@
+openRanges-both.chpl:6: In function 'testRanges':
+openRanges-both.chpl:10: error: parallel iteration is not supported over unbounded ranges
```

Comment:
Am I understanding this right? We don't support that iteration, so we get promotion, which gives us this error.

This is upsetting :(

Reply:
That's right.  This is a bit like saying `(1..10) < 3` ([TIO](https://tio.run/##S85ILEjN@f@/vCizJDUnT0PDUE/P0EDTxljT@v9/AA)) except with an open range as well `(..) < 3`.

Note that this behavior hasn't changed in this PR or release.  I added these tests because I was expecting to lock in that these forms currently result in a syntax error, but got this instead.  I agree it could be confusing, particularly for the user who hopes that our support of `lo..<hi` implies we also support `lo<..<hi` or `lo<..hi`.  But since it's an error (rather than a working program), I didn't sweat it too much since it means that things wouldn't be backward-breaking if we changed it later.

Also, note that there are lots of similar expressions one could write that could get them into similar trouble like `lo..==3`.

Reply:
I see. I still wish we could do something about these. But good to know that this is not something we are adding new.